low-level/files/opt/seagate/sspl/setup/resource_map/resource_map.py

```diff
@@ -67,3 +68,24 @@ def get_health_template(uid, is_fru: bool):
                 "specifics": []
             }
         }
+
+    @staticmethod
+    def set_health_data(health_data: dict, status, description=None,
```

Comment:
Check if something similar doesnt exists. Update that if its exists and if needed

Reply:
yes, this is there in multiple PR's, still kept it so that code will be working, this will get merged with Shriya's PR I guess.